front/app/component-library/components/Accordion/index.tsx

```diff
@@ -142,25 +142,29 @@ const Accordion = ({
           <ChevronIcon name="chevron-right" />
         </TitleButton>
       </Box>
-      <CSSTransition
-        in={isExpanded}
-        timeout={timeoutMilliseconds}
-        mountOnEnter={true}
-        unmountOnExit={true}
-        exit={false}
-        classNames={`expanded`}
+      <Box
+        role="region"
+        aria-live="polite"
+        id={`collapsed-section-${uuid}`}
+        aria-labelledby={`accordion-title-${uuid}`}
       >
```

Comment:
The region needs to always be present in the DOM, and the CSSTransition meant it was being mounted/unmounted. I just added a Box wrapper and set the correct aria attributes there instead.